_posts/2016-11-20-algoritms4-chapter1.md

```diff
@@ -94,15 +111,158 @@ Java basic structure --> Standard I/O:
 > * Don't omit anyone step!! 
 >   Following: declaring, creating and initializing.
 
+> * The name of the array present the entire array. The array is a quotative type. 
+> If you want to copy an array, what you should do is to declare, create, and 
+> initialize a new array and then copy all the values to the new array. See task3.
```

Comment:
Array is initialized with a sequential of memory. Once it is created, one can only change its content, but cannot change its size.